### ISSUE TITLE
About rounding the balance number 

### ISSUE THREAD

**Issue by zhangzhongnan928** [AlphaWallet/alpha-wallet-ios#1873]
For example, if the real balance is 1.537356754, the display balance should be 1.5373 instead of 1.5374

reason:
1. don't set a wrong expectation, users will expect they can get the full display balance available to use   
2. since we don't have a "send maximum balance" function, users may manually key in the number display in balance, they will key in 1.5374 which will trigger "insufficient balance" or if it is a contract call, will result in transaction fail (I did many times😢）

**Comment by hboon** [comment]
@zhangzhongnan928 yes, we do that for the BC project for the same reason. All balances are 4 decimal places.

**Comment by zhangzhongnan928** [comment]
should do the same with any balance displayed in TokenScript action card @James-Sangalli

**Comment by hboon** [comment]
They *might* be moving to 5 decimal places soon.
